docs/settings/subscription.md

```diff
@@ -1,11 +1,13 @@
 # Subscription
 
-Subscription is the same as a custom calendar configuration, except that there is no query and url are added.
+Subscription is the same as a custom calendar configuration, except that there is no query and url are added. 
```

Comment:
Since the current custom calendar integration page is pretty much blank I wonder if there is a better way to describe this.   Maybe something like "A Subscription to another calendar allows the events on that calendar to show up in logseq..."

In all honesty, I'm actually not sure what you can do with a calendar subscription.  Based on the video overview of the plugin https://www.youtube.com/watch?v=m2RFQHmA-zA it seems like events from a shared calendar will show up in the calendar view of the plugin.  Will they also show up as tasks on the journal page?   Is there a way to create new events on the subscribed calendar FROM logseq?

Reply:
Thank you for your advice. I will add the necessary instructions or documentation links to the settings page later.